assets/javascripts/discourse/connectors/after-admin-user-fields/custom-user-fields.gjs

```diff
@@ -37,6 +40,21 @@ export default class CustomUserFields extends Component {
     </AdminFormRow>
 
     {{#if @outletArgs.buffered.has_custom_validation}}
+      <AdminFormRow
+        @label="discourse_authentication_validations.value_validation_regex.label"
+      >
+        <Input
+          @value={{@outletArgs.buffered.value_validation_regex}}
+          class="value-validation-regex-input"
+        />
```

Comment:
to avoid using the old Input component: (you'll need to add some imports)

```suggestion
        <input
          {{on "input" (withEventValue (fn (mut @outletArgs.buffered.value_validation_regex)))}}
          value={{@outletArgs.buffered.value_validation_regex}}
          type="text"
          class="value-validation-regex-input"
        />
```